templates/less/components/form.less

```diff
@@ -1,20 +1,4 @@
-.formContainer {
-  width: 290px;
-  height: 280px;
-  margin: 0 auto;
-  @media (min-width : 360px) and
-  (max-width : 450px){
-    width: 320px;
-  }
-  @media (min-width : 451px) {
-    width: 400px;
-    height: 250px;
-  }
-  border-radius: 7px;
-  padding-top: 25px;
-  margin-bottom: 30px;
-  background-color: white;
-
+form {
```

Comment:
this feelds wrong. Why `form` instead of `.form`? Are we using forms again instead of divs?